compiler/src/test/java/com/squareup/anvil/compiler/dagger/TestClassInUpperCasePackage.kt

```diff
@@ -0,0 +1,4 @@
+@file:Suppress("PackageDirectoryMismatch", "PackageName")
+
+package com.uppercase.Package
+class TestClassInUpperCasePackage
```

Comment:
Nit: Add a space between package and class and comment like `// These classes are used in some of the unit tests. Don't touch them.`. Even though it's unconventional, please put the class into the right directory.